portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,176 @@
 <!DOCTYPE html>
 <html>
   <head>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>My Portfolio 1</title>
+
+    <link href="https://fonts.googleapis.com/css2?family=Lato:ital@1&display=swap" rel="stylesheet">
+    
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <nav> 
+          <span>Navbar</span>
+            <a href="#box">Color Picker</a>
+            <a href="#gallery">Gallery</a>
+            <a href="#aboutContent">About</a>
+      </nav>
+
+    <div class="hero"> 
+        <div class="hero-text"> 
+            <h1>Sofia Montana</h1>
+            
+        </div>
     </div>
+
+    <div id="aboutContent"> 
+        <h2 style="font-size: 36px;">About Me... </h2>
```

Comment:
Put the styles into the css file

Reply:
ok I will change it . The reason why I did this is because I only wanted to change the font size so I thought it would be a little unnecessary to have this in the css.